classes/mcPy/McPy.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import multiprocessing
+import os
+import sys
+
+from .Parser import Parser
+import classes.Server as Server
+
+
+def get_available_core():
+    try:
+        avail_cores = len(os.sched_getaffinity(0))
+    except AttributeError:
+        # Fix for windows, which doesnt support getaffinity
+        logging.warning(
+            "Falling back to multiprocessing cpu_count to calc cores. Most likely getaffinity is not supported on "
+            "your OS")
+        avail_cores = multiprocessing.cpu_count()
+    return avail_cores
+
+
+def _launch(parser: Parser):
+    try:
+        logging.info("The Blackfire probe")
+        # noinspection PyUnresolvedReferences
+        from blackfire import probe  # Profiler: https://blackfire.io free with the Git Student Package
+    except ImportError:
+        BLACKFIRE_ENABLED = False
+        logging.info("Blackfire not installed: passing")
+    else:
+        BLACKFIRE_ENABLED = True
+        probe.initialize()
+        probe.enable()
+        logging.info("Enabled!")
+
+    avail_cores = get_available_core()
+
+    # Here starts the server :D
+    server = Server.Server(parser, avail_cores)
+    if parser.test:
+        server.run_test()
+        return
+    server.start()
+
+    # End Network stuff
+    logging.info("Shutting down server!")
```

Comment:
server.stop() is already printing a similar message, either remove it from there, or vice-versa

Reply:
Done